setup.py

```diff
@@ -14,10 +18,17 @@
 # python3.4 setup.py bdist_wheel upload
 # python3.4 -m pip install
 
+here = os.path.dirname(__file__)
+
+def read(fname, encoding='utf-8'):
+    with io.open(os.path.join(here, fname), encoding=encoding) as f:
+        return f.read()
+
 setup(
     name='vstruct',
     version='2.0.2',
     description='Vivisect Structure Definition/Parsing Library',
+    long_description=read('README.rst'),
```

Comment:
In the specific case of a Python package readme, I'd say you lose almost nothing.

Anything that Markdown can be made to do reST can too. Particularly with the excellent Pandoc package. In the world of Python though docutils and sphinx have you covered.

The caveats as I know them:

 - In the wider world Markdown is more popular. So more collaborators will know it.
 - Github's web editor only knows Markdown, so *in the editor* you don't get a formatting toolbar. Preview still does the right thing.
 - It's another syntax to remember.
 - Aesthetic preference for one syntax over the other.